src/modules/esl-event-listener/core/targets/inert.target.event.ts

```diff
@@ -0,0 +1,43 @@
+import {overrideEvent} from '../../../esl-utils/dom/events/misc';
+
+/**
+ * Event names that could be triggered by {@link ESLWheelTarget}
+ */
+export type WheelEventName = 'longwheel';
+
+/**
+ * Describes scroll information provided with {@link ESLWheelTarget}
+ */
+export interface ESLWheelEventInfo {
```

Comment:
Add an axis property to have information which axis exactly produces the event
Dispatch separate events for x and y axises